tsdb/engine/tsm1/cache.go

```diff
@@ -21,10 +21,12 @@ import (
 const ringShards = 128
 
 var (
-	ErrCacheInvalidCheckpoint = fmt.Errorf("invalid checkpoint")
```

Comment:
This error has been unused since b0fb8a0a27d5380f7255275939c3881801caeb3e (Dec 2015).